flexibleengine/resource_flexibleengine_cce_node_v3_test.go

```diff
@@ -65,20 +65,23 @@ func TestAccCCENodeV3_data_volume_encryption(t *testing.T) {
 		CheckDestroy: testAccCheckCCENodeV3Destroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccCCENodeV3_data_volume_encryption(rName),
+				Config: testAccCCENodeV3_volumes_encryption(rName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckCCENodeV3Exists(resourceName, clusterName, &node),
 					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttrPair(resourceName, "root_volume.0.kms_key_id",
+						"flexibleengine_kms_key_v1.test", "id"),
 					resource.TestCheckResourceAttr(resourceName, "data_volumes.#", "1"),
 					resource.TestCheckResourceAttrPair(resourceName, "data_volumes.0.kms_key_id",
 						"flexibleengine_kms_key_v1.test", "id"),
 				),
 			},
 			{
-				ResourceName:      resourceName,
-				ImportState:       true,
-				ImportStateVerify: true,
-				ImportStateIdFunc: testAccCCENodeImportStateIdFunc(),
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateIdFunc:       testAccCCENodeImportStateIdFunc(),
+				ImportStateVerifyIgnore: []string{"tags"},
```

Comment:
can we delete this line as the issue of `tags` was fixed?

Reply:
good catch, should be good now